src/main/java/de/terrestris/momo/service/MomoPasswordResetTokenService.java

```diff
@@ -139,6 +140,7 @@ protected E buildConcreteInstance(User user, Integer expirationTimeInMinutes) {
 	 * @throws URISyntaxException
 	 * @throws UnsupportedEncodingException
 	 */
+	@Transactional(readOnly = true)
```

Comment:
This method `generate and save the unique reset-password token for the user`. Is it safe to use readonly here?

Reply:
Nice catch! I think you're right. I removed the annotation again.